src/snyk/common/configuration/configuration.ts

```diff
@@ -79,6 +80,8 @@ export class Configuration implements IConfiguration {
   private readonly defaultSnykCodeBaseURL = 'https://deeproxy.snyk.io';
   private readonly defaultAuthHost = 'https://snyk.io';
   private readonly defaultOssApiEndpoint = `${this.defaultAuthHost}/api/v1`;
+  private readonly defaultBaseApiHost = 'https://api.snyk.io';
```

Comment:
What is the difference between `https://snyk.io/api/v1` API and API under `https://api.snyk.io`?

Reply:
I don't know, it's out of my scope.

Reply:
Please clarify this, since we need to understand the context.

Reply:
@sangress any update on this?

Reply:
You need to check with gloo team probably.